.github/workflows/build-kata-static-tarball-amd64.yaml

```diff
@@ -24,6 +24,7 @@ jobs:
           - kernel-gpu
           - kernel-gpu-snp
           - kernel-gpu-tdx-experimental
+          - kernel-sev
```

Comment:
Yeah, having some of the things for SNP named `snp` and some of those using `sev` is confusing, but it's what it's at this point.